prov/cxi/include/cxip.h

```diff
@@ -1271,6 +1274,9 @@ struct cxip_req {
 	uint64_t trig_thresh;
 	struct cxip_cntr *trig_cntr;
 
+	/* pointer to the shared receive entry */
```

Comment:
I don't care enough about this to make you fix and repush but if you do another update, probably drop this comment. I think it's unnecessary

Reply:
removed.